### ISSUE TITLE
Missing numpy when installing from PyPI

### ISSUE THREAD

**Issue by gdurif** [getkeops/keops#113]
Hi,
I get the following error when trying to install `pykeops` from PyPI in a clean Python virtual environment:
```
$ pip install pykeops
Collecting pykeops
  Downloading https://files.pythonhosted.org/packages/a4/0d/faf4fbffa0fa325b72e3f4fc26c41ae70a2dede16f97f2dcf093897411f3/pykeops-1.4.1.tar.gz (319kB)
    100% |████████████████████████████████| 327kB 2.7MB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-yxxk10wu/pykeops/setup.py", line 11, in <module>
        from pykeops import __version__ as current_version
      File "/tmp/pip-build-yxxk10wu/pykeops/pykeops/__init__.py", line 35, in <module>
        from .test.install import test_numpy_bindings, test_torch_bindings
      File "/tmp/pip-build-yxxk10wu/pykeops/pykeops/test/install.py", line 1, in <module>
        import numpy as np
    ModuleNotFoundError: No module named 'numpy'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-yxxk10wu/pykeops/

$ python -V
Python 3.6.9

$ uname -a
Linux xxxxx 5.4.0-52-generic #57~18.04.1-Ubuntu SMP Thu Oct 15 14:04:49 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by gdurif** [comment]
Note: local install from master works
```
$ pip install .
Processing /xxxxx/keops
Collecting numpy (from pykeops==1.4.1.1)
  Cache entry deserialization failed, entry ignored
  Downloading https://files.pythonhosted.org/packages/a6/fc/36e52d0ae2aa502b211f1bcd2fdeec72d343d58224eabcdddc1bcb052db1/numpy-1.19.4-cp36-cp36m-manylinux1_x86_64.whl (13.4MB)
    100% |████████████████████████████████| 13.4MB 135kB/s 
Installing collected packages: numpy, pykeops
  Running setup.py install for pykeops ... done
Successfully installed numpy-1.19.4 pykeops-1.4.1.1
```

**Comment by bcharlier** [comment]
fixed in 1af2124
